### PR TITLE
feat: replace home page text

### PR DESCRIPTION
**Pull request by tisidro**
FSA22V2-288

### Description:

Change placeholder text to actual intro text on home page.

### Spec:

Designs: [Designs](https://www.figma.com/file/UjMQEwUvAsnFAMhYNdUiVT/Apprenticeship-Capstone?node-id=244%3A2183&t=A0sL00y3exNBMMfW-0)

See Story: [FSA22V2-ISSUE](https://sparkbox.atlassian.net/browse/FSA22V2-288?atlOrigin=eyJpIjoiNmU5YTRlNGU5NzVkNDlmNzllOTNhMzAzNTI0YWMyZWQiLCJwIjoiaiJ9)

### Validation:

- [ ] This PR has code changes, and our linters still pass.
- [ ] This PR affects production code, so it was browser tested (see below).
- [ ] This PR has new code, so new tests were added or updated, and they pass.

#### To Validate:

1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, etc).
2. Pull down all related branches.
3. Run `npm install` to make sure you have all proper dependencies. (this project requires Node 16+)
4. Copy the `.env.example` file, making sure it's in the root of your project, and rename it `.env.local`. Search for "Accessible Components" in 1Password to find the API Key and Base ID. Add both of those secrets to your newly created `.env.local` file. That should establish your connection to our Airtable database.
5. Run `npm run lint` to confirm no errors.
6. Run `npm run test` to confirm all tests pass.
7. In terminal: `npm run dev`.
8. Check [localhost:3000](http://localhost:3000/) to see changes.
9. Run axe Devtools on affected pages to confirm no urgent errors.

The text you see on the home page should match the intro copy on the card :)

### Pending Questions:

1.
2.

---

#### Browser Testing

In these browsers, behavior & design closely match original specifications. A user is able to access all content and functionality, including the usability of required assistive devices, such as keyboard and screenreader.

**macOS**

- [ ] Safari (last 2 major versions)
- [ ] Chrome (last 6 months)
- [ ] Firefox (last 6 months)

**Windows**

- [ ] Chrome (last 6 months)
- [ ] Firefox (last 6 months)
- [ ] Edge (last 6 months)

**Mobile**

- [ ] Safari (last 2 major versions)
- [ ] Chrome on Android (last 6 months)
- [ ] Firefox on Android (last 6 months)

### CODE REVIEW

**Review by empee3 — DISMISSED** [review]

**Review by jkohrt7 — DISMISSED** [review]
LGTM!

File: src/pages/index.page.jsx [hunk 1]
```diff
@@ -11,10 +11,11 @@ export default function Home() {
...
-                Intro copy goes here, such as: Build accessible components more
-                accurately and in less time with these easy-to-read cheatsheets
-                based on WCAG and ARIA best practices. Check back often as
-                we&#39;re adding new components frequently!
+                Building accessible components is hard: the requirements are
+                complicated, the specifications are dense, and good examples are
+                sparse. These cheatsheets should make that a little bit
+                easier—and they&#39;re based on WCAG and ARIA best practices.
+                Check back often as we&#39;re adding new components frequently!
               </p>
```
Comment (dustin-jw) [inline src/pages/index.page.jsx:19]: Let's split this into two paragraphs. The content should be good, though!

```suggestion
<p className="home-top__top-description">
  Building accessible components is hard: the requirements are
  complicated, the specifications are dense, and good examples are
  sparse.
</p>
<p className="home-top__top-description">
  These cheatsheets should make that a little bit
  easier—and they&#39;re based on WCAG and ARIA best practices.
  Check back often as we&#39;re adding new components frequently!
</p>
```

Comment (tisidro) [reply src/pages/index.page.jsx]: Thanks Dustin, I did the paragraph break!

Comment (dustin-jw) [reply src/pages/index.page.jsx]: Is `home-top__top-description` doing anything? I just looked at the deploy preview and that class doesn't appear to apply any styles. We can probably remove the `className` if that's the case.

Comment (tisidro) [reply src/pages/index.page.jsx]: I checked, and it's doing nada--so it's gone now. Thanks for spotting it!

**Review by dustin-jw — DISMISSED** [review]

**Review by dustin-jw — APPROVED** [review]

**Pull request merged**